server/modules/category/manager/categoryManager.js

```diff
@@ -159,11 +159,7 @@ export const updateCategoryProp = async (user, surveyId, categoryUuid, key, valu
         t
       ),
     ])
-    const categories = await validateCategories(surveyId, t)
-    return {
-      categories,
-      category: R.prop(categoryUuid, categories),
-    }
+    return validateCategory(surveyId, categoryUuid, t)
```

Comment:
I like to put the params as an object to avoid future problems with the order

Reply:
this is an old module, I didn't want to do too many changes in this pull request. It needs to be refactored a bit, but maybe later in another pull request